### PR TITLE
Carefully explained the checksum problem.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Explains why we need requirement for selected representation

## Note

I somewhat repeated part of the introduction to explicitly
clarify the requirement and make this document independent
from the reading of RFC3230

### CODE REVIEW

**Comment by ioggstream** [comment]
Hi @LPardue

here's the artifact for this build. Is that good enough to share it with the others?

- https://19-183264553-gh.circle-artifacts.com/0/tmp/artifacts/draft-polli-resource-digests-http.html#rfc.section.2

Thx++,
R

File: draft-polli-resource-digests-http.md [hunk 1]
```diff
@@ -196,20 +196,81 @@ The definitions "representation", "selected representation", "representation dat
...
+Request:
+
+    PUT /entries/1234 HTTP/1.1
+    Content-Type: application/json
+
+    H4sIAItWyFwC/6tWSlSyUlAypANQqgUAREcqfG0AAAA=
+
+~~~
 
+A Range-Requests alters the payload body, conveying a partial representation.
```
Comment (LPardue) [inline draft-polli-resource-digests-http.md]:
```suggestion
A Range-Request alters the payload body, conveying a partial representation.
```

File: draft-polli-resource-digests-http.md [hunk 2]
```diff
@@ -196,20 +196,81 @@ The definitions "representation", "selected representation", "representation dat
...
+
+~~~
+Request:
+
+    GET /entries/1234 HTTP/1.1
+    Accept-Range: bytes=1-7
```
Comment (LPardue) [inline draft-polli-resource-digests-http.md]:
```suggestion
    Range: bytes=1-7
```

**Review by LPardue — CHANGES_REQUESTED** [review]
Minor nits on the range request but other it LGTM!

**Pull request merged**